lib/main.dart

```diff
@@ -144,6 +142,25 @@ class _TenTenOneState extends State<TenTenOneApp> {
         GoRoute(
             path: '/',
             builder: (BuildContext context, GoRouterState state) {
+              final tentenoneChangeNotifier = context.watch<TenTenOneChangeNotifier>();
+              if (!tentenoneChangeNotifier.isReady()) {
+                Timer(const Duration(milliseconds: 750), () {
+                  // delay removing the splash screen as otherwise the screen will jump due to loading the png
+                  FlutterNativeSplash.remove();
+                });
```

Comment:
Damn, the transition is almost perfect, but somehow the size of the replacement splash screen is slightly different.

Reply:
yeah, I don't know why though, I am using the exact same image as for the splash screen.